src/Uno.Toolkit.UI/Extensions/SelectorExtensions.cs

```diff
@@ -22,7 +22,7 @@ namespace Uno.Toolkit.UI
 	/// Extensions for <see cref="Selector"/>
 	/// </summary>
 	[Bindable]
-	public static class SelectorExtensions
+	public static partial class SelectorExtensions
```

Comment:
We should probably just merge these two SelectorExtension files together instead of separating them. The split between "Behaviors" and "Extensions" is kind of a mess right now in the solution. Thoughts @Xiaoy312?

Reply:
It is a mess especially. However, I do believe that they needs to be kept separated.
- `Behaviors\`: for attached properties
- `Extensions\`: for c# extensions

That being said, this case here should be brought to `Behaviors\` and have the two merged. Attached properties has no place here.